tekton/README.md

```diff
@@ -105,7 +108,8 @@ To run the `publish-tekton-pipelines` `Task` and create a release:
      value: gs://tekton-releases # REPLACE with your own bucket
    ```
 
-4. Setup the required credentials for the `release-right-meow` service acount,
+4. To run against your own infrastructure (not needed for the actual releases),
+also setup the required credentials for the `release-right-meow` service acount,
```

Comment:
i wonder if it would be clearer to make a separate section about "running against your own infrastructure" so when you're doing an official release you can ignore these instructions more easily and see exactly what you  need to do?

Reply:
NIT: s/acount/account